docs/breadcrumbs/overview.md

```diff
@@ -0,0 +1,188 @@
+---
+endpoints:
+  breadcrumb_generate:
+    name: Generate Breadcrumb
+    url: /api/v1/breadcrumb/generate
+    method: GET
+    description: Generate a specific breadcrumb type for the target bird.
+    params:
+      - name: auth_token
+        required: true
+        type: string
+        description: A valid auth token. Also supports read-only [Flock API keys](http://localhost:8080/flocks-settings/flock-auth-token.html).
+      - name: node_id
+        required: true
+        type: string
+        description: A valid node_id
+      - name: kind
+        required: true
+        type: string
+        description: Name of desired breadcrumb type. Please check "[Available Breadcrumb Kinds](/breadcrumbs/overview.html#available-breadcrumb-kinds)"
+                     for the different Breadcrumb kind values.
+    response: JSON structure with output content of the crumb generation.
+  breadcrumb_download:
+    name: Download Breadcrumb
+    url: /api/v1/breadcrumb/download
+    method: GET
+    description: Download specific breadcrumb type for a target bird, or an archive of all possible breadcrumbs.
+    params:
+      - name: auth_token
+        required: true
+        type: string
+        description: A valid auth token. Also supports read-only [Flock API keys](http://localhost:8080/flocks-settings/flock-auth-token.html).
+      - name: node_id
+        required: true
+        type: string
+        description: A valid node_id
+      - name: kind
+        required: false
+        type: string
+        description: Name of desired breadcrumb type. Please check "[Available Breadcrumb Kinds](/breadcrumbs/overview.html#available-breadcrumb-kinds)"
+                     for the different Breadcrumb kind values. If not specified, an archive of all possible breadcrumbs will be returned.
+    response: Temporary download link or archive of different breadcrumbs.
+---
+
+# Overview
+
+<APIEndpoints :endpoints="$page.frontmatter.endpoints" :path="$page.regularPath">
+
+::::: slot details
+
+Breadcrumbs are a powerful tool for guiding attackers towards your Canaries. They take the form of profiles or device entries across various different services that an attacker might poke at for vulnerable hosts, leading them directly towards the Canary. 
+
+Breadcrumbs are built based on the enabled services of a specific target bird and are templated with the various settings and parameters configured for that service.
```

Comment:
> configured for that service.

I think "those services" instead of "for that service" may work better hear, seeing as we're using a plural when services are initially mentioned in the sentence.